Containerfile

```diff
@@ -0,0 +1,30 @@
+FROM quay.io/devto/ruby:2.7.1
```

Comment:
What're the main differences between this and the official Docker images? The distribution?

Reply:
It's a tailored ruby container to fit our needs which will give us more control over our stack.

Reply:
Do we own it? Is it open source?

Reply:
Yes, we own that and yes I plan on open sourcing the Containerfile that made it later once I get the other repos in order.